docs/mobile-apps/automated-testing/appium/real-devices.md

```diff
@@ -39,6 +39,83 @@ The following application file types are supported for real device tests:
+
+Since W3C and JWP use different formats for specifying your test capabilities, it is important to make sure you configure your tests accurately so your intended protocol is followed and your settings are applied correctly.
+
+### How Sauce Labs Determines Your Protocol
+
+When Sauce Labs executes your test configuration, it looks for the presence of certain indicators in the session creation request to determine whether it should apply the JWP or W3C protocol. The following table outlines how Sauce Labs evaluates your creation request.
+
+|Indicator|Determination|
+|:---|:---|
+|`sauce:options` node is present within the capabilities node.|W3C|
+|`desiredCapabilities` node is absent.|W3C|
+|`desiredCapabilities` node is present AND `sauce:options` node is absent.|JWP|
+
+### Examples of JWP and W3C Configurations
+
+The main difference between JWP and W3C is the format required to specify your test capabilities. In JWP, all capabilities are defined within the `desiredCapabilities` node, while in W3C, all capabilities are defined under the `capabilities` node within the `firstMatch` property. Each capability uses a namespace to indicate one of the following three categorizations:
+
+* WebDriver standard capabilities have no namespace
+* Appium specific capabilities require the prefix `appium:`
+* Sauce Labs custom capabilities are set within the `sauce:options` node
+
+The following examples illustrate this difference in the respective specifications.
+
+<Tabs
+  groupId="protocol"
+  defaultValue="jwp"
+  values={[
+    {label: 'JWP', value: 'jwp'},
+    {label: 'W3C', value: 'w3c'},
+  ]}>
+
+<TabItem value="jwp">
+
+```
+"desiredCapabilities": {
+    "platformName" : "android";
+    "app" : "./apps/mda-1.0.6-8.apk";
+    “deviceName" : "Samsung.*Galaxy.*”;
+    “orientation” : “portrait”;
+    “platformVersion" : "8.1";
+    "appiumVersion" : "1.21.0";
+ }
+```
+
+</TabItem>
+<TabItem value="w3c">
+
+```
+"capabilities": {
+    "firstMatch": [
+      {
+        "platformName" : "android",                    #standard capability
+        "appium:app" : "./apps/mda-1.0.6-8.apk";       #Appium capabilities
+        “appium:deviceName" : "Samsung.*Galaxy.*”;
+        “appium:orientation” : “portrait”;
+        “appium:platformVersion" : "8.1";
+        "sauce:options" : {                           #Sauce custom capabilities
+           "appiumVersion" : "1.21.0";
+           "sessionCreationRetry" : "2",
+           "sessionCreationTimeout" : "300000",
+           "name" : "MobileWebsiteTest (w3c)"
+        }
+      }
+    ]
+}
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+You can avoid having to add the `appium:` prefix to Appium specific capabilities by upgrading your [Appium client library](http://appium.io/docs/en/about-appium/appium-clients/) to a version that automatically applies the prefix or installing the [relaxed-caps plugin](https://github.com/appium/appium-plugins/tree/master/packages/relaxed-caps).
```

Comment:
I love this note, but would also like to add something like

> ... Our recommendation is to start using and getting used to the `appium:` prefix to prevent confusion in the future

Reply:
makes sense to me.